### PR TITLE
Add and remove

### PR DESCRIPTION
**Pull request by DanteAlonsoHT**
<h1>To Do list: Add and Remove</h1>

## [LINK DEMO](https://raw.githack.com/DanteAlonsoHT/To_Do_List_webpack/add-and-remove/dist/index.html)

In this project, I've been working on adding Local Storage as the local database for the web app, including more dynamic functionalities in order to make a better application to improve the UX. I've implemented a CRUD system for this web app.

I've been working on the next files:

- src/index.js
- src/styles.css
- src/index.html
- src/checkbox.js
- src/crud.js

Finally, I got approval from linters.

### CODE REVIEW

File: src/checkbox.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 export class Methods {
-  addTask() {
+  static addTask() {
     let inputTask = ''; let lastIndexTasks; let newTask;
     inputTask = document.getElementById('input-task').value;
     const formTasks = document.getElementById('form-tasks');
-    if (inputTask.length > 8 && inputTask.length < 32) {
+    if (inputTask.length > 8 && inputTask.length < 64) {
       // newTask = [[inputTask, 'false']];
       lastIndexTasks = String(this.getLocalStorage().length);
```
Comment (Lameck1) [inline src/checkbox.js]: - [x] Please let us meet this requirement before we can proceed. Look at the quote below 👇🏼:

> By default new tasks should have the property completed set to false and the property index set to the value of the new array length (i.e. if you're adding a 5th task to the list, the index of that task should equal to 5).

### Explanation
_**Adding a task for the first time:**_
- New array length will be `1`, therefore the index property of this task should be `1`.
- Currently, the index property is starting from 0. Please work on this.
Further explanation is in the bracketed part of the above quote. It says the `5th` task should have `index` property of `5`. That only means that the first task started off with `index` property of `1`.

### Consider
```JavaScript
lastIndexTasks = String(this.getLocalStorage().length + 1);
```

OR

```JavaScript
lastIndexTasks = String(this.getLocalStorage().length);
newTask = [{ task_name: inputTask, completed: 'false', index: lastIndexTasks + 1 }];
```

**Review by Lameck1 — CHANGES_REQUESTED** [review]
Hi @DanteAlonsoHT , 

Good job so far!
There is something small that we need to look into before we can move ahead.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/checkbox.js (hunk 1 above)
Comment (DanteAlonsoHT) [reply src/checkbox.js]: Thanks for the explanation!

**Review by hemantso — APPROVED** [review]
Hi @DanteAlonsoHT , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 

Alt Text

Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**